packages/dso-toolkit/components/Componenten/input-number/input-number.njk

```diff
@@ -6,12 +6,13 @@
   {% endif %}
   {% render '@button', {type: 'button', modifier: 'dso-tertiary', label: minusButtonLabel, icon: 'minus-square', iconOnly: 'true', disabled: minusButtonInactive} %}
   <input type="number"
-          id="{{ id }}"
-          readonly
-          tabindex="-1"
-          {% if min %} min="{{ min }}"{% endif %}
-          {% if max %} max="{{ max }}"{% endif %}
-          class="dso-input-step-counter"
-          value="{{ count }}">
+    id="{{ id }}"
+    readonly
+    tabindex="-1"
+    {% if min or min == 0 %} min="{{ min }}"{% endif %}
```

Comment:
Kan je niet met `{% if min >= 0 %}` iets doen? En dan hetzelfde met het attribuut `max` ? Ervan uitgaande dat we nooit onder 0 willen aanbieden, natuurlijk :)

Reply:
Tja, stel je vraagt om temperatuur in graden Celcius, dan is het min `-273`. Dat moet wel mogelijk zijn. Ik baal er ook van dat  een `0` waarde van `min` hier als falsy wordt geevalueerd en daardoor dan dus niet doorkomt. Maar mbt. `max` -> daar moet eigenlijk ook diezelfde check op idd. Want anders kun je geen `max="0"` krijgen.